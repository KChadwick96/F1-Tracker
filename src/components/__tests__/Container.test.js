import React from 'react';
import renderer from 'react-test-renderer';
import { Text } from 'react-native';

import Container from '../Container';

it('renders without crashing', () => {
  const rendered = renderer.create(
    <Container>
      <Text>Hello</Text>
      <Text>There</Text>
    </Container>
  ).toJSON();
  expect(rendered.children.length).toEqual(2);
});
