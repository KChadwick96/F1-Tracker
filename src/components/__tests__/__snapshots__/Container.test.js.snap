// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders its children 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "yellow",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    There!
  </Text>
</Component>
`;
